rita_client/src/traffic_watcher/mod.rs

```diff
@@ -96,14 +96,13 @@ pub async fn query_exit_debts(msg: QueryExitDebts) {
     }
     let gateway_exit_client = is_gateway_client();
     let start = Instant::now();
-    let exit_addr = msg.exit_internal_addr;
     let exit_id = msg.exit_id;
     let exit_port = msg.exit_port;
     // actix client behaves badly if you build a request the default way but don't give it
     // a domain name, so in order to do peer to peer requests we use with_connection and our own
     // socket specification
     let our_id = settings::get_rita_client().get_identity();
-    let request = format!("http://{}:{}/client_debt", exit_addr, exit_port);
+    let request = format!("http://[{}]:{}/client_debt", exit_id.mesh_ip, exit_port);
```

Comment:
I don't think this ends up doing anything. We're paying the exit internal ip (so the one inside wg_exit) so there's no chance of this being the wrong ip. You could make the argument that using the mesh_ip lets us pay before wg_exit is online, but we need internet access to pay, so it doesn't allow us to make payments when we otherwise couldn't and also isn't any better about not paying the wrong exit.

Reply:
From what I understand all exits in the cluster have the same internal ip 172.168.0.254? So in that case if traffic is routed through a specific 2602 addr, how do query_exit_debts know which exit 172.168.0.254 refers to?

Reply:
It will always be the exit they are currently connected to because that's what wg_exit is pointed at.